src/node/txdownload_impl.cpp

```diff
@@ -425,4 +425,15 @@ std::pair<bool, std::optional<PackageToValidate>> TxDownloadImpl::ReceivedTx(Nod
 
     return std::make_pair(true, std::nullopt);
 }
+
+bool TxDownloadImpl::HaveMoreWork(NodeId nodeid)
```

Comment:
If the wrapper is named `HaveMoreWork`, maybe have `GetTxToReconsider` be called `GetMoreWork`? Otherwise they seem incongruous.

Reply:
I called it `HaveMoreWork` since its purpose is to tell `ThreadMessageHandler()` if there is more work to do (doesn't matter what kind of work):
https://github.com/bitcoin/bitcoin/blob/3c0b66c2ece7695ab59f466d7b4d81a4c18ffd76/src/net_processing.cpp#L5083 
https://github.com/bitcoin/bitcoin/blob/3c0b66c2ece7695ab59f466d7b4d81a4c18ffd76/src/net.cpp#L2902-L2903

`GetTxToReconsider`'s user is peerman, in `ProcessOrphanTx`.

I think scheduled reconsideration work could become more generic in the future, i.e. include packages. But I think  `ProcessOrphanTx` is probably more readable if the function name specifies we're getting 1 transaction to reconsider.